cypress/jenkins/cypress.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -x
+set -e
+
+pwd
+ls -al
+cd "dashboard"
+
+yarn install
+yarn add --dev -W mocha mochawesome mochawesome-merge \
+  mochawesome-report-generator cypress-multi-reporters \
+  mocha-junit-reporter 
+
+NO_COLOR=1 cypress run --browser chrome \
+  --reporter cypress-multi-reporters \
+  --reporter-options \
+    configFile=cypress/jenkins/reporter-options.json \
+  --env grepTags=CYPRESSTAGS
```

Comment:
think this needs to be `GREP_TAGS`. https://github.com/rancher/dashboard/blob/master/cypress.config.ts#L99. 

Tests now run as either an admin or a standard user. This env will ensure tests designed to run only against admin or only against standard user run. If standard user is selected you need to also supply `TEST_USERNAME` as well. @yonasberhe23 can walk you through those requirements

Reply:
CYPRESSTAGS is just a placeholder to dinamically replace it by the `GREP_TAGS` values.

All those values are read and supplied in the Corral Package `run.sh`
https://github.com/rancherlabs/corral-packages/pull/17/files#diff-0691477a86647ab7e92bdc8f54ff53460a79660fbacdbf2f5fc175ef09c97cc4